### ISSUE TITLE
About states colors

### ISSUE THREAD

**Issue by KipK** [KipK/openevse-gui-v2#13]
On the old UI, I thought there was too much state color displayed around the status bar, make it confusing.
To keep it clear for users, I propose to keep only 3 color states:

- Disabled ( red ) 
- 
![image](https://user-images.githubusercontent.com/488789/203312781-486dd86b-0e33-46c6-b2c9-ed2dbed27142.png)

- Active ( not charging ) ( green ) 

![image](https://user-images.githubusercontent.com/488789/203312686-51661bcf-f753-451d-9164-0499003d5497.png)

- Charging ( yellow )
- 
When charging, box-shadow around status bar animates in fading,  all bolts Icons on page will display also in Yellow.

![OpenEVSE WiFi](https://user-images.githubusercontent.com/488789/203312506-0d777568-fdb8-48a7-a57d-1d86bb528e32.gif)

Any thoughts on it ? 
@jeremypoulter
@fhteagle

**Comment by jeremypoulter** [comment]
The colours around the status match the LCD/LED colours which is important for the US market

**Comment by KipK** [comment]
Do we have to keep that on new UI? 

I mean, using blue for bolt icon , won't be really en phase

**Comment by jeremypoulter** [comment]
I don't think we have to keep the colours in the UI, but there is a level of consistency with the LCD/LED (which are mandatory) if we do

**Comment by fhteagle** [comment]
I really do not have a lot of preference about this. I personally rarely look at the physical display of the unit, and could not tell you what color means what on the display. So if even a very detail oriented person like me does not care, well......

Having the color meaning be consistent between the GUI and the display is nice on principle of course.

**Comment by KipK** [comment]
@glynhudson @chris1howell could we consider changing the lcd led colors to more explicit ones?
I mean, Red -> Error is ok, but I'd rather have yellow -> charging, blue -> sleeping, green -> active and would match new UI better.

**Comment by chris1howell** [comment]
LCD colors are limited.

We have 3 LEDs that can be ON or OFF

So we have
R0 G0 B0 backlight off
R1 G0 B0 RED (ERROR)
R1 G1 B0 PURPLE (SLEEP)
R1 G0 B1 YELLOW (Connected)
R0 G1 B0 GREEN (Ready)
R0 G1 B1 CYAN (Charging)
R0 G0 B1 BLUE (very dark not used)
R1 G1 B1 WHITE

I believe we have to use Green for ready, Red for error and blue for
charging but I will have to look at the specs.

**Comment by KipK** [comment]
I've changed few things on state colors in UI to be consistent with leds.

Green, available, Yellow connected, fading from yellow to cyan blue while charging , red error states.

Also, @chris1howell I've put back the old logo. It's awful but let start with it for now. :)
That should be all good in latest gui_v2 daily build
